ads/google/adsense.js

```diff
@@ -41,6 +41,9 @@ export function adsense(global, data) {
 
   const i = global.document.createElement('ins');
   i.setAttribute('data-ad-client', data['adClient']);
+  if (data['adChannel']) {
+    i.setAttribute('data-ad-channel', data['adChannel']);
+  }
```

Comment:
given the repeating pattern, shall we do a for-loop?
you will need to introduce a `camelCaseToDash` function in `string.js`.